src/renderNothing.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function () {
+  const major = React.version.split('.')[0];
```

Comment:
you are checking version of React that you import yourself, it's not going to change on the fly